rector-config-docroot-dir.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Core\Configuration\Option;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(__DIR__ . '/vendor/palantirnet/drupal-rector/config/drupal-8/drupal-8-all-deprecations.yml');
```

Comment:
Does this need to be updated from `.yml` to `.php`?

I think we'll want to also add commented out lines for the individual files in case people want to pick and choose.

Reply:
Yes, if it's a Rector config.